src/components/TextMaskedInput/TextMaskedInput.js

```diff
@@ -163,6 +181,8 @@ TextMaskedInput.PUBLIC_PROPS = {
   labelWeight: PropTypes.string,
   labelClasses: PropTypes.string,
   maxLength: PropTypes.number,
+  iconPrefix: PropTypes.string,
```

Comment:
@MayUWish same callout as above. Please add description and expected values for these two properties.

Reply:
added. Thank you.